website/src/css/custom.css

```diff
@@ -1722,8 +1722,9 @@ section > h2:not(.resource-section) {
   padding-right: 3rem;
   padding-left: 3rem;
   padding-top: 0rem;
-  padding-bottom: 6.25rem;
-  margin-bottom: 0;
+  padding-bottom: 0;
+  margin-bottom: 6.25rem;
+  box-shadow: inset 30px 0 30px -30px rgba(0, 0, 0, 0.3), inset -30px 0 30px -30px rgba(0, 0, 0, 0.3);
```

Comment:
I never seen this before so I am not that confident if this is the right  for css.   
The box shadowing works when I tested it. Welp,  I learned something new today.

Reply:
Also I'm seeing the same think in dark mode:

<img width="363" alt="Screenshot 2023-03-17 at 9 58 34 AM" src="https://user-images.githubusercontent.com/72401230/225926233-e5393685-4b02-4817-b6ad-ca6d9c099039.png">

I noticed you said you added a transparent gradient on the Swiper carousel, so I'm wondering if this is intentional? I'm not sure I like the look of it, though.

Reply:
@JKarlavige Oh yeah, I was going to mention getting the style from the Figma comp, but then I realized there wasn't an actual Figma comp for this (I don't think?)